### ISSUE TITLE
Eurotronic Spirit Zigbee Dropped off network

### ISSUE THREAD

**Issue by Morphy99** [Koenkk/zigbee2mqtt#6317]
This may be related to [https://github.com/Koenkk/zigbee2mqtt/issues/4571](url) but the error reported by the OP is different.
I've had no problems with my Eurotronic TRV's for well over a year until the other day, one of them dropped off the network partially. I keep getting device announcements in the log from it and it doesn't respond to anything:
`Publish 'set' 'system_mode' to 'TRV Bedroom' failed: 'Error: Read 0x00158d0001ffd623/1 hvacThermostat([16392], {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":4151,"transactionSequenceNumber":null}) failed (Timeout - 37663 - 1 - 12 - 513 - 1 after 10000ms)'`

I do recall having to reset it a couple of months ago as it seemed to have lost the calibration to close the valve completely when off. I've looked at the firmwares of all 3 of mine and they're all:
Firmware build date
20191014
Firmware version
22190930

Running on the latest CC2530 source routing coordinator fw 20201128 Z2M 17.1

**Comment by Morphy99** [comment]
So I've tried removing this and re-joining but now I can't control temperature. Pressing reconfigure in Z2M has resolved this and it appears to function correctly.

**Comment by Morphy99** [comment]
This has happened again but this time I didn't notice and it caused more serious problems with the whole network going down in the middle of the night. Looking through the logs I can occasional device_announcements so I'm guessing this is normal? Or is there something up with my network?
I can see the TRV started spamming device_announce yesterday at 9;25 but the network seems fine till about 3:55 when everything drops off. I tried restarting Z2M a few times this morning which worked a for a few minutes till everything went down again. At which point I removed the batteries from the TRV as it kept joining even though join was disabled and I had force removed it. I also powered off the system and reflashed my cc2530 with the latest source routing fw. No idea if that will help but I was on 20201128 anyway although it was in beta when I flashed it last so not sure if there's been any changes.

**Comment by Morphy99** [comment]
Noticed it was back doing it again today so I've looked further and read a comment about it could be WiFi interference which has led me to think I just found the culprit. Another system I've been testing something on has a CC2530 in it and even though the serial connection to the RPi GPIO wasn't enabled it was still powered and must have been causing interference! I've disconnected it now and removed and rejoined the TRV again so keeping fingers crossed it stays online now.
